docs/blogs/kubecon_cfp.md

```diff
@@ -0,0 +1,45 @@
+# ElasticDL: Kubernetes-native Deep Learning System with Elastic Scheduling
+
+## Description
+
+In these years, Kubernetes becomes the fundamental infrastructure of machine
+learning. Kubeflow project provides some operators to run distributed machine
+learning jobs on Kubernetes. It uses gang scheduling policy. For example, a job
+requires N pod, it will keep pending if there are N-1 or less available pods. And
+during execution, the job will crash if any one of these N instances crashes. As
+a result, the job pending time is long and the resouce utilization is insufficient.
+
+ElasticDL implements elastic scheduling policy. A job doesn't need wait for all
+the required resource ready and start promptly. A job process can be preempted
+by a higher priority job, join freely once the resource become sufficient later
+and the entire job keeps running. It can significantly shorten the job waiting
+time and improve the resource utilization.
+
+## Benefits to the Ecosystem
+
+This talk introduces a system ElasticDL, it implements elastic scheduling policy
```

Comment:
ElasticDL boosts the cluster utilization up to 90%, on on-premise clusters at Ant Financial and on Google Cloud, as It makes full use of residual resources to run deep learning jobs.  Moreover, it enables the running of deep learning jobs in lower priority than online services on the same cluster.  The master senses and uses resource left by online services.

Reply:
Good suggestion, updated according to the comments.